web/src/main/java/org/openmrs/module/web/ModuleResourcesServlet.java

```diff
@@ -90,20 +92,19 @@ protected File getFile(HttpServletRequest request) {
 		}
 		
 		String relativePath = ModuleUtil.getPathForResource(module, path);
-		String realPath = getServletContext().getRealPath("") + MODULE_PATH + module.getModuleIdAsPath() + "/resources"
-		        + relativePath;
+		Path p = Paths.get(getServletContext().getRealPath(""), MODULE_PATH, module.getModuleIdAsPath(), "resouces",
```

Comment:
Please remove changes from this file for now. I will need to take a closer look at what we are doing here so we do not introduce a bug.